luthier/config/src/metrics.lua

```diff
@@ -60,8 +60,13 @@ So the metric "difference" is the (rather silly) formula:
             on directly, if any, only applies for nested metrics
 --]]
 -------------------------------------------------------------------------------
+local M = {}
 
-return {
+local aggregation_metrics = {
+    longSumCO = {
+        maker = "longSum",
```

Comment:
I'm thinking we would be better off calling this `type` rather than `maker`. It makes the
configuration more symmetrical with the rest of our config. Plus, the fact that we're using
"makers" in Fili is an implementation detail, and it isn't going to be immediately obvious to
people what a "maker" is, or why that's how we specify the type of metric this is.

Reply:
Disregard this. I forgot that people configure makers in Luthier as well.